component.yaml

```
@@ -1,13 +1,13 @@
 platform: Common
 buildPlatform:
 displayName: Testinium
-description: "Run your test plans with Testinium"
+description: "Enables uploading mobile applications to Testinium and running the selected test plan."
 inputs:
 - key: "AC_TESTINIUM_APP_PATH"
   defaultValue: "$AC_TESTINIUM_APP_PATH"
   isRequired: true
   title: Path of the build
-  description: "Full path of the build. For example $AC_EXPORT_DIR/Myapp.ipa."
+  description: "Full path of the build file. Example for iOS: `$AC_EXPORT_DIR/MyApp.ipa`. Example for Android: `$AC_APK_PATH`."
```

In this [PR](https://github.com/appcircleio/appcircle-workflow-components/pull/532/files#diff-4c7400e9cf1b139e68055a616bebf6f24aba8efcdbfb3680ea7b678fa12d5789) we decided user should use this step after Xcodebuild for Devices, but the `$AC_EXPORT_DIR/MyApp.ipa` path will be generated after Export Build Artifact. We need to change the path with `$AC_OUTPUT_DIR/MyApp.ipa`. Because this path will be generated after xcodebuild for devices step.
```suggestion
  description: "Full path of the build file. Example for iOS: `$AC_OUTPUT_DIR/MyApp.ipa`. Example for Android: `$AC_APK_PATH`."
```